org.chromium.Chromium.yaml

```diff
@@ -154,7 +162,7 @@ modules:
   - name: extensions
     buildsystem: simple
     build-commands:
-      - mkdir -p /app/chromium/{extensions,native-messaging-hosts,policies}
+      - mkdir -p /app/chromium/{{unified-,}extensions,native-messaging-hosts,policies/{managed,recommended}}
```

Comment:
But now I see the second half of this hunk and I wonder whether the policies bit works in the currently-released version!

Reply:
Ah yes, #78, so no, it doesn't. Unfortunately `org.chromium.Chromium.Policy` is not the one being discussed above :-)